packages/common/src/server.ts

```diff
@@ -73,12 +73,20 @@ export type YogaServerOptions<TAdditionalContext, TRootValue> = {
     | {
         typeDefs: TypeSource
         resolvers?:
-          | IResolvers<TRootValue, TAdditionalContext & YogaInitialContext>
+          | IResolvers<
+              TRootValue,
+              TUserContext & TServerContext & YogaInitialContext
```

Comment:
Shouldn't it be UserContext?

Reply:
Context factory given by the user passed to useExtendContext which merges the initial context with the returned object. So resolvers receive an merged object that contains the initial context and user context